client/src/components/Card/Card.Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Looks like Card body footer and header are very similar with some different styles, could you refactor this into a single component and pass some aditional props?